internal/kafka/client_test.go

```diff
@@ -313,3 +313,75 @@ func TestNewSaramaClient_TLS(t *testing.T) {
 		require.Error(t, tryConnect(configtls.ClientConfig{}))
 	})
 }
+
+func TestNewSaramaConsumerGroup_RebalanceAndInstanceId(t *testing.T) {
```

Comment:
The two configuration settings are independent, so should we test them independently too?

Reply:
Right, but is it important that they are tested together? It seems to me we could make this one just about testing GroupRebalanceStrategy